### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by ruimashita**
Add pull request template.

## Motivation and Context

In order to improve productivity, in order to each reviewer approve PR, in order to developer create new PR,
The PR template helps us to easy understand what the PR is, how effective the PR is, how affects the PR is, etc.., considering about each direction of PR reviewer and PR developer.
related issues: #25

## Description

Please see.
https://raw.githubusercontent.com/blue-oil/blueoil/006f1fd7cfc3880ec715e3b56076c0793d9f5ccf/.github/PULL_REQUEST_TEMPLATE.md
Also, I created this PR following new template.

## How has this been tested?

I tested that this PR merged my forked repo is work well.
You can see how it works in below link.
https://github.com/ruimashita/blueoil/compare/master...add-augmentor-support

## Screenshots (if appropriate):
When someone create new pull request
screen shot 2018-12-10 at 21 36 03

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by tkng** [comment]
Thanks, LGTM as a starter.

**Comment by lm-konda** [comment]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Motivation and Context
+<!--- 💭💡💭 Why is this change required? What problem does it solve? 💭💡💭 -->
+<!--- 🔗 If it fixes an open issue, please link to the issue here. 🔗 -->
+
+## Description
+<!--- 📝🎯📝 Describe your changes in detail. 📝🎯📝 -->
+
+## How has this been tested?
+<!--- 🙆👌🙆 Please describe in detail how you tested your changes. 🙆👌🙆 -->
+<!--- Include details of your testing environment, details of your manually tests, snippet code or commands of your manually tests, table of experiments results metrics, tests ran to see how your change affects other areas of the code, etc. -->
```
Comment (lm-lily) [inline .github/PULL_REQUEST_TEMPLATE.md]: I am not so sure about this, is it supposed to be "manual tests" instead of "manually tests"?

**Comment by n-nez** [comment]
Thank you! Looks good. :100: :+1: 
But I wonder if all the sections are required? :thinking: 
For light-weight PRs (something you spent a few hours to write VS something you spend one month+ working in), writing it all seems to be a bit overkill... :sweat_drops:

**Comment by tkng** [comment]
@n-nez I think we can skip to fill it for light-weight PRs, e.g. the purpose of #17 is obvious, we don't need too long description for such a PR.

**Comment by n-nez** [comment]
@tkng LGTM then :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ruimashita) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks.

**Pull request merged**